cob_default_robot_config/cob3-9/command_gui_buttons.yaml

```diff
@@ -46,15 +46,6 @@ group5:
     [home,move,home],
     [folded,move,folded],
     [pregrasp,move,pregrasp]]
-group6:
```

Comment:
why is this section removed from dashboard? It is necessary on all cob3s to move the arm to the frontside of the robot.